### PR TITLE
Update quickstart.md

### PR DESCRIPTION
**Pull request by agibsonccc**
Add overview

## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

## Quick checklist

The following checklist helps ensure your PR is complete:

- [ ] Reviewed the [Contributing Guidelines](https://github.com/deeplearning4j/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.
- [ ] Ran mvn formatter:format (see [formatter instructions](http://code.revelc.net/formatter-maven-plugin/examples.html#Setting_Source_Files) for targeting your specific files).

### CODE REVIEW

**Comment by bkanui** [comment]
Hi Francois,

I see some simple edits that can be applied. I’ve never reviewed through GitHub. So, I do not know how, and might not have permissions, to make or suggest changes. LMK if there is something I need to do through the Review Request.

So, I’m including my suggestions here, preceded with >>  All the edits are trivial. 

Thanks,
Bethany


We recommend that you join our [Gitter Live Chat](https://gitter.im/deeplearning4j/deeplearning4j). Gitter is where you can request help and give feedback, but please do use this guide before asking questions we've answered below. If you are new to deep learning, we've included [a road map for beginners](./deeplearningforbeginners.html) with links to courses, readings and other resources. If you need an end to end tutorial to get started (including setup), then please go to our [getting started](http://deeplearning4j.org/gettingstarted)
>> ending period missing

+Overview

+===================================

+Deeplearning4j is a "pick and choose with everything available" framework. That means *everything* is made available for you from the beginning. Deeplearning4j has several sub projects for you to build end to end applications.

+Deeplearning4j is not tensorflow or pytorch. See [our comparison page](https://deeplearning4j.org/compare-dl4j-torch7-pylearn)
>> ending period missing

+If you're coming here looking to just deploy models to production, you might like our [model import from keras](http://deeplearning4j.org/keras)
>> ending period missing

+Deeplearning4j has several submodules. This ranges from a visualization UI to distributed training on spark. For an overview of these modules, please look at our [examples](https://github.com/deeplearning4j/dl4j-examples)
>> These range from
>> ending period missing

+To get started on a simple desktop app, you need 2 things. An [nd4j backend](http://nd4j.org/backend.html) and deeplearning4j-core. For more code, see the [simpler examples submodule](https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/pom.xml#L64)
>> ending period missing

+If you want a super flexible deep learning api, there are 2 ways you can go.  You can use nd4j standalone See our [nd4j examples](https://github.com/deeplearning4j/dl4j-examples/tree/master/nd4j-examples) or the [computation graph api](http://deeplearning4j.org/compgraph)
>> API
>> You can use nd4j standalone.  (missing period)
>> ending period missing

+If you want to do distributed training on spark, you can see our [spark page](http://deeplearning4j.org/spark]()
>> you can see our  >>> see our  (delete you can)
>> ending period missing. 
>> Are the closing parens () needed?

+Keep in mind though, we will not setup spark for you. If you want to setup distributed spark and gpus that is largely up to you. Deeplearning4j simply deploys as a jar file on an existing spark cluster.

+If you want spark with gpus, we recommend [spark with mesos](https://spark.apache.org/docs/latest/running-on-mesos.html).

+If you want to deploy on android, you can see our [android page](http://deeplearning4j.org/android)
>> android, see out (delete you can)
>> ending period missing

+We deploy optimized code for various hardware architectures natively. We use c++ based for loops just like everybody else.

+See our [c++ framework](https://github.com/deeplearning4j/libnd4j)

+Deeplearning4j has 2 other notable components. [Baked in hyper param search](https://github.com/deeplearning4j/Arbiter)
+and [built in ETL](https://github.com/deeplearning4j/DataVec)
>> ending period missing
>> Are these on separate lines? (+) as bullet items?. If not, it’d be ok to combine into one sentence.

+Overall, deep learning4j is meant to be an end to end platform for building real applications. Not just a tensor library with automatic differentiation. If you want that, that's in nd4j and it's called [samediff](https://github.com/deeplearning4j/nd4j/tree/master/samediff)  samediff is still in high alpha yet but if you want to take a crack at contributing please come in to our livechat.
>> ending period missing after parens )
>> still in high alpha, but if (delete yet)

+Lastly, if you are benchmarking deeplearnin4j, please consider coming in to our live chat and asking for help. Deeplearning4j has [all the knobs](http://deeplearning4j.org/native)
>> deeplearning4j (missing g)
>> coming to our  (delete in)
>> ending period missing, unless sentence continues with (but some may..)

+but some may not work like you expect the python frameworks to do. Generally you have to build deeplearning4j from source for some applications.
>> but some might not work the same as python frameworks. (reworked)
>> Generally, for some applications, you have to build deeplearning4j from source. (reworked)

**Comment by bkanui** [comment]
All reviews can be performed through the Github web interface.

https://help.github.com/articles/reviewing-changes-in-pull-requests/

I'm really sorry you invested any time in reviewing anything through email.
I'm afraid Adam will need your comments to be in Github to address them
conveniently.

-- 
FG

File: quickstart.md [hunk 1]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
 
 This is everything you need to run DL4J examples and begin your own projects.
 
-We recommend that you join our [Gitter Live Chat](https://gitter.im/deeplearning4j/deeplearning4j). Gitter is where you can request help and give feedback, but please do use this guide before asking questions we've answered below. If you are new to deep learning, we've included [a road map for beginners](./deeplearningforbeginners.html) with links to courses, readings and other resources. 
+We recommend that you join our [Gitter Live Chat](https://gitter.im/deeplearning4j/deeplearning4j). Gitter is where you can request help and give feedback, but please do use this guide before asking questions we've answered below. If you are new to deep learning, we've included [a road map for beginners](./deeplearningforbeginners.html) with links to courses, readings and other resources. If you need an end to end tutorial to get started (including setup), then please go to our [getting started](http://deeplearning4j.org/gettingstarted)
```
Comment (bkanui) [inline quickstart.md]: ending period missing

File: quickstart.md [hunk 2]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+Overview
+===================================
+
+Deeplearning4j is a "pick and choose with everything available" framework. That means *everything* is made available for you from the beginning. Deeplearning4j has several sub projects for you to build end to end applications.
+Deeplearning4j is not tensorflow or pytorch. See [our comparison page](https://deeplearning4j.org/compare-dl4j-torch7-pylearn)
```
Comment (bkanui) [inline quickstart.md]: ending period missing

File: quickstart.md [hunk 3]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+
+If you're coming here looking to just deploy models to production, you might like our [model import from keras](http://deeplearning4j.org/keras)
```
Comment (bkanui) [inline quickstart.md]: ending period missing

File: quickstart.md [hunk 4]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+Deeplearning4j has several submodules. This ranges from a visualization UI to distributed training on spark. For an overview of these modules, please look at our [examples](https://github.com/deeplearning4j/dl4j-examples)
```
Comment (bkanui) [inline quickstart.md]: This ranges from > These range from 
ending period missing

File: quickstart.md [hunk 5]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+To get started on a simple desktop app, you need 2 things. An [nd4j backend](http://nd4j.org/backend.html) and deeplearning4j-core. For more code, see the [simpler examples submodule](https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/pom.xml#L64)
```
Comment (bkanui) [inline quickstart.md]: ending period missing

File: quickstart.md [hunk 6]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+If you want a super flexible deep learning api, there are 2 ways you can go.  You can use nd4j standalone See our [nd4j examples](https://github.com/deeplearning4j/dl4j-examples/tree/master/nd4j-examples) or the [computation graph api](http://deeplearning4j.org/compgraph)
```
Comment (bkanui) [inline quickstart.md]: api > API
You can use nd4j standalone. (missing period)
ending period missing also after ) end of sentence.

File: quickstart.md [hunk 7]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+If you want to do distributed training on spark, you can see our [spark page](http://deeplearning4j.org/spark]()
```
Comment (bkanui) [inline quickstart.md]: you can see our > see our (delete 'you can')
ending period missing
Are the closing parens () needed?

File: quickstart.md [hunk 8]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+Keep in mind though, we will not setup spark for you. If you want to setup distributed spark and gpus that is largely up to you. Deeplearning4j simply deploys as a jar file on an existing spark cluster.
+If you want spark with gpus, we recommend [spark with mesos](https://spark.apache.org/docs/latest/running-on-mesos.html).
+
+
+If you want to deploy on android, you can see our [android page](http://deeplearning4j.org/android)
```
Comment (bkanui) [inline quickstart.md]: android, you can see our > android, see our (delete 'you can;)
ending period missing

File: quickstart.md [hunk 9]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+We deploy optimized code for various hardware architectures natively. We use c++ based for loops just like everybody else.
+See our [c++ framework](https://github.com/deeplearning4j/libnd4j)
+
+Deeplearning4j has 2 other notable components. [Baked in hyper param search](https://github.com/deeplearning4j/Arbiter)
+
+and [built in ETL](https://github.com/deeplearning4j/DataVec)
+
+Overall, deep learning4j is meant to be an end to end platform for building real applications. Not just a tensor library with automatic differentiation. If you want that, that's in nd4j and it's called [samediff](https://github.com/deeplearning4j/nd4j/tree/master/samediff)  samediff is still in high alpha yet but if you want to take a crack at contributing please come in to our livechat.
```
Comment (bkanui) [inline quickstart.md]: ending period missing
still in high alpha yet but if > still in high alpha, but if (delete 'yet')

File: quickstart.md [hunk 10]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+
+Lastly, if you are benchmarking deeplearnin4j, please consider coming in to our live chat and asking for help. Deeplearning4j has [all the knobs](http://deeplearning4j.org/native)
```
Comment (bkanui) [inline quickstart.md]: deeplearnin4j > deeplearning4j (missing g)
coming in to our > coming to our (delete to)
ending period missing -- unless the sentence is continued to next line.

File: quickstart.md [hunk 11]
```diff
@@ -18,7 +18,41 @@ Quick Start Guide
...
+but some may not work like you expect the python frameworks to do. Generally you have to build deeplearning4j from source for some applications.
```
Comment (bkanui) [inline quickstart.md]: reworked both phrases/sentences to
but some might not work the same as python frameworks.
Generally, for some applications, you have to build deeplearning4j from source.

Comment (agibsonccc) [inline quickstart.md]: @bkanui you are welcome to push changes to this branch as well. Merge when you feel it's ready. Thanks!

**Pull request merged**